sdk/search/Microsoft.Azure.Search.sln

```diff
@@ -0,0 +1,49 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Our data plane tests rely on our management plane SDK, as well as some utilities defined in our management plane tests. I don't see those projects in this solution. Please make sure that loading and building the solution and running tests in Playback mode all continue to work.

Reply:
I have added management.test that to the solution file